docs/resources/bootstrap_git.md

```diff
@@ -0,0 +1,75 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "flux_bootstrap_git Resource - terraform-provider-flux"
+subcategory: ""
+description: |-
+  Commits Flux components to a Git repository and configures a Kubernetes cluster to synchronize with the same Git repository.
+---
+
+# flux_bootstrap_git (Resource)
+
+Commits Flux components to a Git repository and configures a Kubernetes cluster to synchronize with the same Git repository.
```

Comment:
Should we add a disclaimer here regarding the fact that it is a release candidate and should be used in production with some caution?

Reply:
I think that would be appropriate if you go out with an RC version first. Something like one of these callouts over here maybe? https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_policy_attachment

Reply:
Added a note at the to of the resource docs.